src/lib/migration/main_workflow.rb

```diff
@@ -77,6 +90,10 @@ def run
       }
     }
 
+    # reboot command: reboot in one minute to give the user a chance to finish YaST
+    # and log out, the reboot can be canceled by "shutdown -c" command if needed
+    REBOOT_COMMAND = "shutdown --reboot +1"
```

Comment:
NP: I'd prefer constants like this to be placed at the top of the file. Probably a matter of taste, though.